ui/app/templates/components/pgp-file.hbs

```diff
@@ -23,7 +23,7 @@
         name={{concat "useText-" this.elementId}}
         class="toggle is-success is-small"
         checked={{this.key.enterAsText}}
-        onchange={{action (toggle "enterAsText" this.key)}}
+        onchange={{action "handleToggle"}}
```

Comment:
This change and https://github.com/hashicorp/vault/blob/fdf0365bc5f0844814450157fcc0249cd3fbe33c/ui/app/components/pgp-file.js#L67-L69
are the two additional file changes for this PR compared to the original. When digging into the failure, it seemed like the `toggle` helper no longer updated `this.key`. 
```
  Integration | Component | pgp file: toggling back and forth: Chrome 130.0
renders the textarea on toggle
Expected:
Element [data-test-pgp-file-textarea] exists once

Result:
Element [data-test-pgp-file-textarea] does not exist

Reply:
I did come across the `toggle-action` helper
https://github.com/hashicorp/vault/blob/52c02ae41d765dd3c3a8f1bc950050262a019e3e/ui/lib/kubernetes/addon/components/page/credentials.hbs#L77
and confirmed it still updates `this.clusterRoleBinding`